includes/functions.php

```diff
@@ -873,6 +873,8 @@ function is_port_valid($port, $device) {
     else {
         $valid = 1;
         $if = strtolower($port['ifDescr']);
+		$ifname = strtolower($port['ifName']);
```

Comment:
Can you fix the space formatting here. 4 spaces for indentation.

your other PRs had the same but I only noticed at the end :(